test/bdd/common_steps.go

```diff
@@ -118,6 +118,20 @@ func (d *CommonSteps) setVariable(varName, value string) error {
 	return nil
 }
 
+func (d *CommonSteps) setVariableToCurrentTime(varName string) error {
+	value := time.Now().UTC().Format(time.RFC3339)
+
+	if err := d.state.resolveVarsInExpression(&value); err != nil {
```

Comment:
You don't need this code.

Reply:
> You don't need this code.

value will never have variables so calling resolveVarsInExpression is not necessary